src/main/kotlin/no/nav/familie/ks/sak/api/dto/BehandlingResponsDto.kt

```diff
@@ -31,7 +33,7 @@ data class BehandlingResponsDto(
     val personer: List<PersonResponsDto>,
     val personResultater: List<PersonResultatResponsDto>,
     val utbetalingsperioder: List<UtbetalingsperiodeResponsDto> = emptyList(),
```

Comment:
Gir det mening å legge til listen over `utbetalingsperioder` i denne PR'en? Er ihvertfall det som trengs for at beløp osv skal vises på behandlingsresultat-siden. 

Det som isåfall må endres på er `UtbetalingsperiodeDetalj` hvor vi også trenger feltene `antallTimer` og `prosent` 😄

Reply:
antallTimer sendes allerede til frontend under personResultater, Kan det ikke gjenbrukes?